config/routes.rb

```diff
@@ -1,7 +1,13 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      namespace :merchants do
+        resources :find, only: [:index]
+      end
       resources :invoices, except: [:new, :edit]
+      resources :merchants, except: %i[new edit] do
+        get '/items', to: 'merchants/items#index'
```

Comment:
@jmejia @icorson3 Is this an acceptable way to write this route? We tried using resources, and could not make it happen.

Reply:
@anon0mys @nergdnvlt This is fine...we just want you to be consistent. As you get further in this project, you might need to manually write your routes. Also, something you didn't ask about but needs to be adjusted is

```ruby
namespace :merchants do
  resources :find, only: [:index]
end
```
We want to use resources for objects we CRUD and right now, find is using an object that already exists so instead, you should manually write these routes. Lastly, if we think about how to use our actions, we want to send "find" to a show method and "find_all" to a index method. with how this is set up, the route /find is going to the index.